authprox/apadmin/test.sh

```diff
@@ -34,7 +34,7 @@ BCSetup(){
 	eval $SED
 	[ -z "$BC" ] && exit 1
 
-	testOK ./bcadmin add spawn:authproxAdd -identity $ID
+	testOK ./bcadmin darc rule --rule spawn:authproxAdd --identity $ID
```

Comment:
does that work? it seems the other tests use `-rule` and `-identity` (one minus sign)

Reply:
both forms seem to work

Reply:
but `bcadmin darc rule -help` is saying double minus, that's why I went for it

Reply:
From the cli doc:
```
If you enable the *UseShortOptionHandling, then you must not use any flags that have a single leading - or this will result in failures. For example, -option can no longer be used. Flags with two leading dashes (such as --options) are still valid.
```